randovania/games/planets_zebeth/presets/starter_preset.rdvpreset

```diff
@@ -0,0 +1,112 @@
+{
+    "schema_version": 58,
+    "base_preset_uuid": null,
+    "name": "Starter Preset",
+    "uuid": "4f335b5e-d858-4adc-8794-eb301e81cd14",
+    "description": "Basic preset.",
+    "game": "planets_zebeth",
+    "configuration": {
+        "trick_level": {
+            "minimal_logic": false,
+            "specific_levels": {}
+        },
+        "starting_location": [
+            {
+                "region": "Brinstar",
+                "area": "Corridor 1",
+                "node": "Starting Point"
+            }
+        ],
+        "available_locations": {
+            "randomization_mode": "full",
+            "excluded_indices": []
+        },
+        "standard_pickup_configuration": {
+            "pickups_state": {
+                "Bombs": {
+                    "num_shuffled_pickups": 1
+                },
+                "Screw Attack": {
+                    "num_shuffled_pickups": 1
+                },
+                "Varia Suit": {
+                    "num_shuffled_pickups": 1
+                },
+                "Hi-Jump Boots": {
+                    "num_shuffled_pickups": 1
+                },
+                "Ice Beam": {
```

Comment:
![grafik](https://github.com/randovania/randovania/assets/38186597/d545f964-f0c9-40d3-b860-782111821214)
Ice beam seems weird. this should look like this instead:
![grafik](https://github.com/randovania/randovania/assets/38186597/fd11628c-0381-416e-9025-bed62b2f78e3)

Reply:
what if you want to start with ice beam?

Reply:
thats unrelated. Starter preset doesnt start with ice beam, and thus should default to "2 shuffled copies".
if you want to *modify* starter preset to start with ice beam,then you can of course always change it to "custom".

Reply:
if you do like on the second screenshot it shuffles only one ice beam

Reply:
that sounds like you have a mistake in your implementation? echoes, dread and msr do this for some items, and it does shuffle all of them.

Reply:
Was a gui bug, will be fixed later.